apps/docs-website/docs/vscode-support.md

```diff
@@ -38,10 +38,18 @@ For [Astro](https://astro.build/) projects, add the astro extension too:
 
 ## Avoid "source.organizeImports"
 
-Sheriff already handle imports sorting, so you have to disable thew VSCode automatic import sorting feature to avoid conflicts:
+Sheriff already handle imports sorting, so you have to disable VSCode automatic import sorting feature to avoid conflicts:
```

Comment:
I think this is slightly misleading. We should probably reword this as: "Sheriff already handle imports sorting, so if you happen to have enabled the VSCode automatic import sorting feature, you should disable it to avoid conflicts"

Reply:
nit: "handles", not "handle"

Otherwise, that makes sense.